daml-lf/spec/transaction.rst

```diff
@@ -386,6 +400,11 @@ As of version 5, this new field is required to be non-empty:
 
 Every element of ``actors`` is a party identifier.
 
+.. note:: *This section is non-normative.*
+
+  Actors are manually by the user invoking fetching the contract -- or
+  in other words, they are _not_ a property of the contract itself.
```

Comment:
I don't understand this sentence. I think a verb is missing.

Reply:
done